### ISSUE TITLE
What is the meaning of score_gt and score_cand?

### ISSUE THREAD

**Issue by CaffreyR** [r-three/t-few#16]
What is the meaning of score_gt and score_cand? How do I better run the model by observing these parameters?

**Comment by muqeeth** [comment]
bs here in the code is batch size I think.

**Comment by CaffreyR** [comment]
But what does it mean for score_sand & gt @muqeeth

**Comment by HaokunLiu** [comment]
score_cand and score_gt means the average score for wrong answers and correct answers.

**Comment by PastelBelem8** [comment]
@HaokunLiu imagine I'd like to persist the scores as [probabilities](https://github.com/r-three/t-few/blob/master/src/models/EncoderDecoder.py#L218), is it safe to assume that `torch.exp(score_gt) + torch.exp(score_cand) < 1`?

**Comment by HaokunLiu** [comment]
> @HaokunLiu imagine I'd like to persist the scores as [probabilities](https://github.com/r-three/t-few/blob/master/src/models/EncoderDecoder.py#L218), is it safe to assume that `torch.exp(score_gt) + torch.exp(score_cand) < 1`?

Ha, you found this issue. In fact, if we are going to compute a probability distribution over all the choices (including correct and incorrect), they should be considered as -logits rather than probabilities. They correspond to $ - \beta (x, y)$ from eq. 2 in the paper.
